content/en/docs/concepts/configuration/secret.md

```diff
@@ -502,13 +502,6 @@ the kubelet on each node to authenticate to that repository. You can configure
 _image pull secrets_ to make this possible. These secrets are configured at the Pod
 level.
 
-The `imagePullSecrets` field for a Pod is a list of references to Secrets in the same namespace
-as the Pod.
-You can use an `imagePullSecrets` to pass image registry access credentials to
-the kubelet. The kubelet uses this information to pull a private image on behalf of your Pod.
-See `PodSpec` in the [Pod API reference](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
```

Comment:
This line is better than the one it replaces:
```markdown
See the [PodSpec API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core) for more information about the `imagePullSecrets` field.
```

For the paragraph we **keep**, could we change that to:
```markdown
See `PodSpec` in the [Pod API reference](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
```
?

Reply:
Hi @jckhmmz .
I think @sftim is suggesting to update the link under the heading `Using imagePullSecrets` with the following sentence:

```
See `PodSpec` in the [Pod API reference](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
```

Do you agree with that suggestion?

Reply:
Hi @jckhmmz ,
If you're still working on this PR, please take a look at @sftim's [suggestion](https://github.com/kubernetes/website/pull/36703#discussion_r966817935)